packages/trader/src/RangeTrader.ts

```diff
@@ -0,0 +1,171 @@
+const { ConvertDecimals, createFormatFunction, createObjectFromDefaultProps } = require("@uma/common");
```

Comment:
This is the key logic for the RangeTrader that executes trades based on a range movement of a synthetic price.